sdk/eventhub/Azure.Messaging.EventHubs.Shared/src/Processor/PartitionLoadBalancer.cs

```diff
@@ -204,7 +204,8 @@ protected PartitionLoadBalancer()
             // partitions it owns.  When an event processor goes down and it has only expired ownership, it will not be taken into consideration
             // by others.  The expiration time defaults to 30 seconds, but it may be overridden by a derived class.
 
-            var utcNow = DateTimeOffset.UtcNow;
+            var utcNow = GetDateTimeOffsetNow();
```

Comment:
Nice.   This is long overdue.

Reply:
Ever think of adding a small Date/Time utility to `Azure.Core` for an abstraction?   Normally, we'd use something like `NodaTime` for this if third party dependencies wasn't a concern.

Reply:
It crossed my mind but because it's not something that we would want customers to ever use shipping it as a public type was never worth it.